src/headers.js

```diff
@@ -167,9 +167,9 @@ export default class Headers extends URLSearchParams {
 		return value;
 	}
 
-	forEach(callback) {
+	forEach(callback, thisArg) {
 		for (const name of this.keys()) {
-			callback(this.get(name), name);
+			callback.call(thisArg, this.get(name), name, this);
```

Comment:
This allows `callback` to be an object with a `call` property, which isn't what we want. Can we instead use `Reflect.apply`?

Reply:
sure! good call 👍